File: Tp Soap/Q1EnJava/src/Reserve/Reservation.java
package Reserve;

import java.util.ArrayList;

public class Reservation {
	public ArrayList reservations = null;

	public Reservation() {
	
		this.reservations = new ArrayList();
	}
	public Boolean reserver(Client c){
		if(!reservations.contains(c)){
			reservations.add(c);
			return true;
		}else{
			return false;
		}
	}
	

}

File: Tp Soap/Q1/Q1/Reservation.cs
using System;
using System.Collections;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Q1

{
    class Reservation
    {
        public ArrayList reservations = null;
       
       
        public Reservation()
        {
            reservations = new ArrayList();
        }

        public Boolean reserver(Client c )
        {
            if (!reservations.Contains(c))
            {
                reservations.Add(c);
                

                return true;
            }
            else
                return false;
        }

        
    }
}
